R/kernel.r

```diff
@@ -362,9 +362,9 @@ run = function() {
 )
 
 comm_manager_env <- new.env()
-#' Get global CommManager instance
+#' Get global Comm_Manager instance
```

Comment:
oh, yes, it’s really called that. missed that one.

that isn’t a naming convention i’ve ever seen in my life. we should rename it to `CommManager`